Source/Core/DolphinWX/Debugger/CodeWindow.cpp

```diff
@@ -28,13 +28,11 @@
 #include "Common/CommonTypes.h"
 #include "Common/StringUtil.h"
 #include "Common/SymbolDB.h"
+#include "Core/ConfigManager.h"
 #include "Core/Core.h"
 #include "Core/Debugger/Debugger_SymbolMap.h"
 #include "Core/Debugger/PPCDebugInterface.h"
 #include "Core/HW/CPU.h"
-#include "Core/HW/Memmap.h"
-#include "Core/HW/SystemTimers.h"
```

Comment:
I just removed them as they're unnecessary, so it was essentially a "clean-up-as-I-go" sort of thing.